ncm-ceph/src/main/pan/components/ceph/v1/schema-osd.pan

```diff
@@ -30,6 +30,8 @@ type ceph_osd = {
     'journal_path' ? string
     'crush_weight' : double(0..) = 1.0
     'labels' ? string[1..]
+    'storetype'? string with match(SELF, '^(blue|file)store$')
```

Comment:
use choice? but this is probably not correct (new entry in old schema)? also panlint fixes and remove next empty line?